clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/preprocessors/AllowLaunchPreProcessor.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.clouddriver.aws.deploy.preprocessors;
+
+import com.netflix.spinnaker.clouddriver.aws.deploy.description.AllowLaunchDescription;
+import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperationDescriptionPreProcessor;
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+public class AllowLaunchPreProcessor implements AtomicOperationDescriptionPreProcessor {
+  @Override
+  public boolean supports(Class descriptionClass) {
+    return AllowLaunchDescription.class.isAssignableFrom(descriptionClass);
+  }
+
+  @Override
+  public Map process(Map description) {
+    // Backwards-compatibility from when AllowLaunch used to overload "account" from the abstract
+    // AWS description.
+    description.put("targetAccount", description.get("account"));
```

Comment:
I'd propose for this to do a putIfAbsent("targetAccount") instead otherwise callers will have to continue to supply account once they have been updated

Reply:
Good one. OK